x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportCloseJobAction.java

```diff
@@ -524,39 +526,75 @@ private void normalCloseJob(ClusterState currentState, Task task, CloseJobAction
             return;
         }
 
+        final Set<String> movedJobs = new HashSet<>();
```

Comment:
Should this set be `Concurrent` since its written to in one thread and read from others?